### DISCUSSION TITLE
Helm installation flexibility

### DISCUSSION THREAD

**Discussion by kamikaze** [bank-vaults/bank-vaults#2282] [Q&A] [answered]
Is there any way to install everything with just helm + regular kubectl apply for CRDs? I mean if I want to install vault into a custom namespace - it would be enough of just adding "-n custom" instead of current cumbersome process.

**Comment by ramizpolic** [comment, marked as answer]
Hi @kamikaze, good question. Due to RBAC configurations and current tooling, it is required to overwrite the default namespace when deploying to a custom namespace in manifests using the documented steps. We are working on making the installation process simpler.
